test/test_in_toto_keygen.py

```diff
@@ -0,0 +1,99 @@
+"""
```

Comment:
In addition to the existing tests, I'm wondering if it'd be nice to have:

1. Create a keypair, and load the public and private keys
2. Create a keypair with different length parameters and load it